docs/Concepts/4.md

```diff
@@ -1,60 +1,152 @@
 
-There's also further research being done in this domain, including the designing of (RLN (Rate Limiting Nullifiers))[./7.md]
+We intend to change this in the future and enable bootstrap by default once we have implemented more decentralized strategies.
 
-### Data confidentiality, Integrity, and Authenticity
+{{< /hint >}}
 
-Confidentiality can be addressed through data encryption whereas integrity and authenticity are achievable through digital signatures.
-These features are provided for in [WAKU-MESSAGE (version 1)](https://rfc.vac.dev/spec/14#version-1) and [WAKU-NOISE](https://rfc.vac.dev/spec/35/) through payload encryption as well as encrypted signatures.
+## Predefined Bootstrap Nodes
 
-> `WAKU-NOISE` enables secure channel negotiation over Waku.
+Addresses of nodes hosted by Status are predefined in the codebase:
 
-## Security Considerations
+https://github.com/status-im/js-waku/blob/e4024d5c7246535ddab28a4262006915d2db58be/src/lib/discovery/predefined.ts#L48
 
-**Lack of anonymity/unlinkability in the protocols involving direct connections including [`13/WAKU2-STORE`](https://rfc.vac.dev/spec/13/) and [`12/WAKU2-FILTER`](https://rfc.vac.dev/spec/12/) protocols**:
-The anonymity/unlinkability is not guaranteed in the protocols like `13/WAKU2-STORE` and `12/WAKU2-FILTER` where peers need to have direct connections to benefit from the designated service.
-This is because during the direct connections peers utilize `PeerID` to identify each other,
-therefore the service obtained in the protocol is linkable to the beneficiary's `PeerID` (which counts as PII).
-For `13/WAKU2-STORE`, the queried node would be able to link the querying node's `PeerID` to its queried topics.
-Likewise, in the `12/WAKU2-FILTER`, a full node can link the light node's `PeerID`s to its content filter.
+**Pros:**
+
+- Low latency
+- Low resource requirements
+
+**Cons:**
+
+- Prone to censorship: node IPs can be blocked
+- Limited: Static number of nodes
+- Poor maintainability: Code needs to be changed to update the list
+
+## Use your own nodes
+
+Developers have the choice to run their own [nwaku](https://github.com/status-im/nim-waku/) nodes
+
+and use them to bootstrap js-waku nodes.
+
+There are two ways to set bootstrap nodes:
+
+1. Using an array of multiaddrs (as `string` or `Multiaddr`):
+
+```ts
+import { Waku } from "js-waku";
+
+const waku = await createNode({
+  bootstrap: {
+    peers: [
+      "/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/443/wss/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm",
+    ],
+  },
+});
+```
+
+2. Passing an async function that returns an array of `multiaddr` (as `string` or `Multiaddr`):
+
+```ts
+import { Waku } from "js-waku";
+
+const waku = await createNode({
+  bootstrap: {
+    getPeers: async () => {
+      const addresses = [];
+
+      // Fetch the multiaddrs from somewhere...
+
+      return addresses;
+    },
+  },
+});
+```
+
+{{< hint info >}}
```

Comment:
Too js-waku specific, just mention that one could decide to pass a list of predefined node as a way to bootstrap their node and list pros and cons for this method.